File: time_convert.py
'''
Challenge
Have the function TimeConvert(num) take the num parameter being passed and return the number of hours and minutes the parameter converts to (ie. if num = 63 then the output should be 1:3). Separate the number of hours and minutes with a colon. 
Sample Test Cases
Input:126
Output:2:6

Input:45
Output:0:45

'''

def TimeConvert(num): 

    hours = num // 60
    mins = num - (hours * 60)
    ans = str(hours) + ":" + str(mins)
    return ans
    
# keep this function call here  
g = input("Enter number minutes: ") 
print(TimeConvert(int(g)))

File: time_convert.rb
# Using the Ruby language, have the function TimeConvert(num) take the num parameter being passed
# and return the number of hours and minutes the parameter converts to (ie. if num = 63 then the
# output should be 1:3). Separate the number of hours and minutes with a colon. 

# Sample Test Cases
# Input:126
# Output:"2:6"

# Input:45
# Output:"0:45"

def TimeConvert(str)

  inTime = str.to_i
  hrs = inTime / 60
  mins = inTime - (hrs * 60)
  
  return hrs.to_s + ":" + mins.to_s
         
end
   
# keep this function call here    
puts TimeConvert(STDIN.gets)